control-plane/api-gateway/binding/route_binding.go

```diff
@@ -177,6 +178,17 @@ func (r *Binder) bindRoute(route client.Object, boundCount map[gwv1beta1.Section
 				})
 			}
 
+			if authFilterReferencesMissingJWTProvider(httproute, r.config.Resources) {
+				results = append(results, parentBindResult{
```

Comment:
this feels weird to have on the parent bind result, but we put the other statuses for external filters there and we don't have really any other place to put a status like this

Reply:
Yep, this is one of the things that I'm not a fan of with the spec, if the status of a route has nothing to do with a particular gateway (though that's how it's related to our controller), you still wind up scoping the status to the given gateway, so a filter missing and multiple parent gateways == multiple statuses saying the same thing, "the filter is missing". We already have this issue with backendRefs too... but frankly it's dumb.

Reply:
Acually, just wondering... is this the status for the filter not having a referenced JWT provider on it? If so, couldn't you just store the status of that missing provider on the filter itself?

Reply:
so we do both: on the route we have a status that just tells the user that one of their auth filters is  referencing an invalid JWT provider and then on the filter itself is where we say what providers are missing

Reply:
Got it, that makes sense. Haven't looked too deeply into what the actual message used on the Route itself is, but I think it'd be fine to do something kind of generic -- like `"filter 'default/myFilter' is invalid"` or something, but I'm cool if it's more specific too

Reply:
yeah right now the message is `externalRef references missing JWT provider` but I can make it a bit more generic

Reply:
updated the message to be more generic